TestFoundation/TestJSONSerialization.swift

```diff
@@ -1373,8 +1373,8 @@ extension TestJSONSerialization {
             outputStream.open()
             let result = try JSONSerialization.writeJSONObject(dict, toStream: outputStream, options: [])
             outputStream.close()
-            if(result > -1) {
-                XCTAssertEqual(NSString(bytes: buffer, length: buffer.count, encoding: String.Encoding.utf8.rawValue), "{\"a\":{\"b\":1}}")
+            if result > -1 {
```

Comment:
Let's leave these diffs out of this PR for clarity.

Reply:
The parentheses here are trivial, but the change in the subsequent line is required for the test to pass as the test itself is buggy. We discussed this in #604 a year ago.